pagopa/pn-frontend
# Fix/pn 5758 group admin should not see contacts

## Short description
Adds check for recapiti item in sidebar and makes recapiti a private route for group admin users in pg-webapp

## List of changes proposed in this pull request
- Adds check for recapiti item in sidebar and hides it for group admin users
- Makes recapiti a private route for group admin users

## How to test
Login pg app with a group admin user, check that recapiti item in sidebar is not visible and route `recapiti` answers with a 'page not found'
